### PR TITLE
feat: make setup

### PR DESCRIPTION
**Pull request by ninabarbakadze**
## Overview

Closes https://github.com/celestiaorg/celestia-zkevm-ibc-demo/issues/7
Closes https://github.com/celestiaorg/celestia-zkevm-ibc-demo/issues/8

## Testing

`docker compose up`
`make setup`

### CODE REVIEW

**Comment by cmwaters** [comment]
Can we rename the directory to `testing/demo/setup` and can we also call this from `make setup` in the `Makefile`

**Comment by ninabarbakadze** [comment]
> Can we rename the directory to `testing/demo/setup` and can we also call this from `make setup` in the `Makefile`

yeah i was gonna this was just a placeholder

File: testing/demo/pkg/setup/initialise_groth16_client.go
Comment (rootulp) [inline testing/demo/pkg/setup/initialise_groth16_client.go:46]: 👍

File: testing/demo/pkg/setup/initialise_sp1tm_client.go
Comment (rootulp) [inline testing/demo/pkg/setup/initialise_sp1tm_client.go:42]: 👍

File: testing/demo/pkg/setup/main.go
Comment (rootulp) [inline testing/demo/pkg/setup/main.go:9]: [no change needed] FYI for other reviewers, this command also creates the channel

**Review by rootulp — APPROVED** [review]
Note: I hit this while testing locally but I can try to debug after this merges, you don't need to block on it:

```
--> Setting up IBC Clients and Channels
Seed: 123ebde147726534
WARNING: proto: file name proto/groth16/v1/groth16.proto does not start with expected celestia/ibc/lightclients/groth16/v1/; please make sure your folder structure matches the proto files fully-qualified names
Initializing the groth16 light client
failed to broadcast the initial client creation message: error occurred while waiting for condition: error in json rpc client, with http response metadata: (Status: 200 OK, Protocol HTTP/1.1). RPC error -32603 - Internal error: tx (5E4F3BE46F290393A640AC60CC934508533AC144B2934C0F4F477B252876814B) not found
exit status 1
make: *** [setup] Error 1
```

**Review by rootulp — APPROVED** [review]

File: testing/demo/pkg/setup/main.go
Comment (ninabarbakadze) [reply testing/demo/pkg/setup/main.go:9]: yeah I wasn't sure how to add that as part of the signature as well. I'll add a godoc actually. can be done in a diff pr.

**Review by cmwaters — APPROVED** [review]

**Pull request merged**
